lib/travis/build/git/clone.rb

```diff
@@ -101,8 +101,8 @@ def checkout
           end
 
           def checkout_ref
-            return 'FETCH_HEAD' if data.pull_request
-            return data.tag     if data.tag
+            return 'FETCH_HEAD'      if data.pull_request
+            return %Q{"#{data.tag}"} if data.tag
```

Comment:
Could we call the `tag` method instead i.e. https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/git/clone.rb#L140-L142 ?